boards/arm/sama5/jupiter-nano/include/board_sdram.h

```diff
@@ -107,7 +107,7 @@
 
 #define BOARD_ADC_PRESCAL          (7)
 #define BOARD_TSD_STARTUP          (40)        /* 40 nanoseconds */
-#define BOARD_TSD_TRACKTIM         (2000)      /* Min 1�s at 8MHz */
+#define BOARD_TSD_TRACKTIM         (2000)      /* Min 1µs at 8MHz */
 #define BOARD_TSD_DEBOUNCE         (10000000)  /* 10 milliseconds (units nanoseconds) */
```

Comment:
can we change to ASCII u?

Reply:
Generally this is my preference, as I believe that source code should only be ASCII.

But when I raised this concern in the mailing list, it seems that people prefer to use such special symbols.

What do you recommend me to do?

Reply:
OK, perfect!

Fixed.